src/Skia/Perspex.Skia.iOS/SkiaView.cs

```diff
@@ -15,40 +15,47 @@
 
 namespace Perspex.Skia.iOS
 {
-    public abstract class SkiaView : GLKView
+    public abstract class SkiaView : UIView //GLKView
     {
-        [DllImport("__Internal")]
-        static extern IntPtr GetPerspexEAGLContext();
+        //[DllImport("__Internal")]
+        //static extern IntPtr GetPerspexEAGLContext();
 
         bool _drawQueued;
-        CADisplayLink _link;
+        //CADisplayLink _link;
         static EAGLContext GetContext()
         {
-            //Ensure initialization
-            MethodTable.Instance.SetOption((MethodTable.Option)0x10009999, IntPtr.Zero);
-            var ctx = GetPerspexEAGLContext();
-            var rv = Runtime.GetNSObject<EAGLContext>(ctx);
-            rv.DangerousRetain();
-            return rv;
+            /* No longer needed with SkiaSharp, but require a variant for HW accel
+                //Ensure initialization
+                MethodTable.Instance.SetOption((MethodTable.Option)0x10009999, IntPtr.Zero);
+                var ctx = GetPerspexEAGLContext();
+                var rv = Runtime.GetNSObject<EAGLContext>(ctx);
+                rv.DangerousRetain();
+                return rv;
+            */
+            return null;
         }
 
 
-        protected SkiaView(Action<Action> registerFrame) : base(UIScreen.MainScreen.ApplicationFrame, GetContext())
+        protected SkiaView(Action<Action> registerFrame) : base(UIScreen.MainScreen.ApplicationFrame)	//, GetContext())
```

Comment:
Again, does this being commented out help anything? Will it be useful in future?

From now on I'll just mark this question as "comments" ;)